packages/table/src/CellDefinitions/ActionCell.tsx

```diff
@@ -1,30 +1,100 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import * as Popper from 'popper.js';
+import { useWindowDimensions } from '@availity/hooks';
+
 import Icon from '@availity/icon';
-import TableActionMenu from '../TableActionMenu';
+import TableActionMenu, { TableActionMenuProps } from '../TableActionMenu';
 import TableActionMenuItem from '../TableActionMenuItem';
 import { Cell, IdType } from '../types/ReactTable';
 import { RecordAction } from '../types/RecordAction';
 import { PrimaryRecordAction } from '../types/PrimaryRecordAction';
+import { UncontrolledTooltip, UncontrolledTooltipProps } from 'reactstrap';
 
 export interface ActionCellConfig<T> {
   actions: RecordAction<T>[];
   primaryAction?: PrimaryRecordAction<T>;
+  tableActionMenuProps?: Omit<TableActionMenuProps, 'children'>;
+  tooltipProps?: UncontrolledTooltipProps;
+  isSticky?: boolean;
 }
 
 const ActionCell = <T extends IdType>({
   actions,
   primaryAction,
+  tableActionMenuProps,
+  tooltipProps,
+  isSticky,
 }: ActionCellConfig<T>): ((cell: Cell<T>) => JSX.Element) => {
   const ActionCellDef = ({ row: { original, id } }: Cell<T>): JSX.Element => {
+    const { height, width } = useWindowDimensions();
+
+    const [menuPositionTransform, setMenuPositionTransform] = useState<string | undefined>();
+
+    const stickyMenuModifiers = useMemo(
+      () => ({
+        eventListeners: {
+          scroll: !isSticky,
+        },
+        maintainInitialPosition: {
+          order: 849,
+          enabled: isSticky,
+          fn: (data: Popper.Data) => {
+            if (!isSticky) {
+              return data;
+            }
+
+            if (!menuPositionTransform) {
+              setMenuPositionTransform(data.styles.transform);
+            } else {
+              data.styles.transform = menuPositionTransform;
+            }
+
+            return data;
+          },
+          phase: 'beforeWrite',
+          requires: ['computeStyles'],
+        },
+      }),
+      [isSticky, menuPositionTransform]
```

Comment:
**QUESTION:** Is `isSticky` needed in the dependency array?

Reply:
Yes - it is utilized here in the memo ( see line 37). This is needed to get the memoized object to update appropriately.